nativelib/src/main/scala/scala/scalanative/runtime/ieee754tostring/ryu/RyuDouble.scala

```diff
@@ -699,21 +699,80 @@ object RyuDouble {
       roundingMode: RyuRoundingMode
   ): String = {
 
-    // Step 1: Decode the floating point number, and unify normalized and
-    // subnormal cases.
-    // First, handle all the trivial cases.
+    // Handle all the trivial cases.
     if (value.isNaN) return "NaN"
     if (value == Double.PositiveInfinity) return "Infinity"
     if (value == Double.NegativeInfinity) return "-Infinity"
     val bits = java.lang.Double.doubleToLongBits(value)
     if (bits == 0) return "0.0"
     if (bits == 0x8000000000000000L) return "-0.0"
 
-    // Otherwise extract the mantissa and exponent bits and run the full
-    // algorithm.
+    val result = new scala.Array[Char](RyuDouble.RESULT_STRING_MAX_LENGTH)
+    val strLen = doubleToChars(value, roundingMode, result, 0)
+
+    new String(result, 0, strLen)
+  }
+
+  @inline
+  private def copyLitteralToCharArray(
```

Comment:
Do you mean the double ''tt" in `copyLitteral` if yes, does it
have a meaning?  Did you mean (consistently) `literal`

Reply:
Oops, sorry for the typo.

It should be `copyLiteralToCharArray`

Reply:
Kind of you to leave a typo so:
1) You know reviewer read that far in the code. Sorta like a college paper.
2) Something for a reviewer to feel good about focusing on the small issues
    and leaving the critical issues of the PR alone.